integration_tests/templates/url_for.html

```diff
@@ -0,0 +1,18 @@
+{# templates/url_for.html #}
```

Comment:
@carrycooldude , this file does not provide any value as you are not using the `url_for` function you implemented.

Reply:
You need to use the `url_for` function here.